demonstrations/tutorial_period_finding.metadata.json

```
@@ -0,0 +1,31 @@
+{
+    "title": "Period finding: A problem at the heart of quantum computing",
+    "authors": [
+        {
+            "username": "mariaschuld"
+        }
+    ],
+    "dateOfPublication": "2025-03-31T10:00:00+00:00",
+    "dateOfLastModification": "2025-03-31T10:00:00+00:00",
+    "categories": [
+        "Getting Started",
+        "Quantum Computing",
+    ],
```

Build error may be due to this trailing comma.

```suggestion
    "categories": [
        "Getting Started",
        "Quantum Computing"
    ],
```